python/ray/dashboard/optional_utils.py

```diff
@@ -261,6 +262,21 @@ def _update_cache(task):
         return _wrapper
 
 
+def deny_browser_requests() -> Callable:
+    """Reject any requests that appear to be made by a browser"""
+    def decorator_factory(f: Callable) -> Callable:
+        @functools.wraps(f)
+        async def decorator(self, req: Request):
+            if browser_detection.is_browser_request(req):
+                return Response(
+                        text="Method not allowed",
+                        status=aiohttp.web.HTTPNotAllowed.status_code,
+                        )
```

Comment:
Should we tell people that browser requests are not supported? Or do we purposely not expose this detail?

Reply:
I don't think there's meaningful value to hiding it although you are so far from a supported path if you wind up there. I can adjust the message.